README.project.md.erb

```diff
@@ -12,50 +12,74 @@ git clone <%= config.git_url %>
-If you are on OSX, install [Docker for Mac](https://docs.docker.com/docker-for-mac/install/).
+#### Prerequisites
 
-If you are on Linux, use your package manager to install `docker`.
+##### General
 
-If you are on Windows, install [Docker for Windows](https://docs.docker.com/docker-for-windows/install/).
+- A GitHub Token
+  - This can be generated at [GitHub](https://github.com/settings/tokens).
+- Access to LastPass
+- Access to AWS
+  - If you do not have access you can request it from support.
+- Tideways API Key
+  - If you wish to run tideways profiling on your development environment, you will find this in LastPass.
 
-You should also install dockrsync from [dockrsync](https://github.com/webtatic/dockrsync).
+##### Docker
 
-#### Project setup
-1. Run `dockrsync setup` and fill in the questions:
-   - The default service should be set as "web".
-   - The anybar question can be skipped over by hitting enter.
-   - We recommend choosing "yes" for the delete flag question.
-2. Copy `docker.env.dist` to `docker.env`
-3. Copy `docker-compose.override.yml.dist` to `docker-compose.override.yml`
-4. Fill in docker.env's secrets - do not commit this file!
+- A working docker setup
+  - On MacOS, use [Docker for Mac](https://docs.docker.com/docker-for-mac/install/)
+  - On Linux, add the official Docker repository and install the "docker-ce" package.
+
+#### Setup
+
+1. Copy `docker.env.dist` to `docker.env`
+2. Fill in docker.env's secrets - do not commit this file!
    - TIDEWAYS_API_KEY can be found in LastPass
    - AWS_ACCESS_KEY_ID is found in your Hem configuration. Check `~/.hem/config.yaml` and `~/.hobo/config.yaml`.
    - AWS_SECRET_ACCESS_KEY should be found wherever AWS_ACCESS_KEY_ID was.
-5. Run the following:
+3. Run the following:
    - for bash/zsh: `eval "$(echo $(printf "%s " $(cat docker.env)) docker-compose build)"`
    - for fish: `eval env (echo (printf "'%s' " (cat docker.env)) docker-compose build)`
-6. Run `docker-compose up -d`
-7. Wait for the build to finish, you can follow along with `docker-compose logs -f web`
-8. Run `docker-compose exec web bash` to log in to your local "web" container.
-9. Run this to let you see the site from a nice domain:
+4. Run this to let you see the site from a nice domain:
     - On Linux/OSX: `echo "127.0.0.1 <%= config.hostname %>" | sudo tee -a /etc/hosts`
     - On Windows, open up `C:\Windows\system32\etc\HOSTS` in your favourite text editor as admin and add `127.0.0.1 <%= config.hostname %>`
-10. Run `dockrsync watch` to watch for changes to your local filesystem and upload them to the web container.
-11. The site will be available at [https://<%= config.hostname %>](https://<%= config.hostname %>). Happy coding!
+5. Setup docker-compose.override config and start docker-compose up:
+
+```bash
+bundle install
+
+rake docker:setup
+docker-compose up -d
+```
+
+or you can use the rake-tasks-docker helper to setup and up in one go via:
 
+```bash
+bundle install
+
+rake docker:up
```

Comment:
This doesn't seem to run `docker-compose up` in a detached state.

Reply:
It also ended in a "Project failed to start" message, despite the web container still building.

Reply:
discussed offline yesterday.

rake docker:up starts in detached and immediately logs. Waits for health checks to pass before giving an indication if the project appears to be working, then closes the log

It's meant as a way to help debug startup issues without forever blocking. If there are no health-checks, it doesn't open the logs and returns immediately.